src/ritika/Asgmt_18_Pg2.java

```diff
@@ -0,0 +1,29 @@
+package ritika;
+/*Assignment-18 : 14th Nov'2020
+Program : 2
+String input = "hmPZ23i9Ws"
+output = "239PZWhmis";
+ */
+public class Asgmt_18_Pg2 {
+	
+	void displayOutput(String input) {
+		String lowerCase = "";
+		String upperCase = "";
+		String digits = "";
+		for (int i=0; i<input.length();i++) {
+			char ch = input.charAt(i);
+			if(Character.isDigit(ch)==true)
+				digits = digits+ch;
+			else if	(Character.isUpperCase(input.charAt(i))==true)
```

Comment:
you should not check == with true, think logically, isUpperCase() already return Boolean value